templates/items/consumable.hbs

```diff
@@ -72,6 +73,18 @@
                 </label>
             </div>
 
+            {{#if (eq system.consumableType "ammo")}}
+            {{!-- Consumable Properties --}}
+            <div class="form-group stacked weapon-properties">
+                <label>{{ localize "DND5E.ItemAmmoProperties" }}</label>
+                {{#each config.physicalWeaponProperties as |name prop|}}
+                <label class="checkbox">
+                    <input type="checkbox" name="system.properties.{{prop}}" {{checked (lookup ../system.properties prop)}}/> {{ name }}
```

Comment:
This was probably done for consistency, but we should try avoid legacy self-closing tags on void elements going forward.

```suggestion
                    <input type="checkbox" name="system.properties.{{prop}}" {{checked (lookup ../system.properties prop)}}> {{ name }}
```